CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/reporting/PluginReportUtilities.java

```diff
@@ -0,0 +1,81 @@
+/*
+* Copyright 2010-2023 Australian Signals Directorate
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.reporting;
+
+/**
+ *
+ * @author capricornunicorn123
+ */
+public class PluginReportUtilities {
+    
+    private PluginReportUtilities() {
+        throw new IllegalStateException("Utility class");
+    }
+    
+    /**
+     * Generates a string containing an integer and the correct plurality of a proceeding noun. 
+     * @param count
+     * @param singularForm
+     * @param pluralForm
+     */
+    public static final String toString(final int count, final String singularForm, final String pluralForm) {
+        if (count == 1) {
+            return count + " " + singularForm;
+        } else {
+            return count + " " + pluralForm;
+        }
```

Comment:
this can probably be a ternary statement (`return count + " " + count == 1 ? singularForm : pluralForm`)